workers/unity/Assets/Fps/Scripts/GameLogic/SkyboxCamera.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+
+public class SkyboxCamera : MonoBehaviour
```

Comment:
why is this needed?

Reply:
Since there are now 2 cameras. 1 rendering just the sky and environment, and 1 rendering the game itself.
This script syncs the rotation and fov of the game camera, so that it feels like its 1 space.